tasks/examples.js

```diff
@@ -93,76 +94,81 @@ async function main() {
     (pkg) => Array.isArray(pkg.examples) && pkg.examples.length !== 0
   );
 
-  await Promise.all(
-    packagesWithExamples.map(async (pkg) => {
-      reporter.info(`Building examples in package \`${pkg.name}\``);
+  // await Promise.all(
```

Comment:
should we leave a note about the commented lines or can they be removed?

Reply:
@emyarod oh duh, thanks for pointing this out! Let me flip back to the old version that runs in parallel